src/registry.js

```diff
@@ -5,6 +5,7 @@ import {retry} from 'rxjs/operator/retry'
 import {publishReplay} from 'rxjs/operator/publishReplay'
 import {httpGet} from './util'
 import assert from 'assert'
+import semverMatch from 'semver-match'
```

Comment:
We could, yeah. It would require more code on our end (filtering which versions satisfies, finding the highest to satisfy). I could do that later today.